app/assets/javascripts/application/messages.js

```diff
@@ -0,0 +1,96 @@
+var mumuki = mumuki || {};
+
+(function (mumuki) {
+
+    MessagesService = {
```

Comment:
`MessageService`? Please use domain names! For example, use `Chat`. 

No `service` or `manager` class will be added to this repo while I am owner of it :stuck_out_tongue: .